fe/fe-core/src/main/java/org/apache/doris/catalog/StorageVault.java

```diff
@@ -120,17 +126,18 @@ public void setId(String id) {
         StorageVaultType type = stmt.getStorageVaultType();
         String name = stmt.getStorageVaultName();
         boolean ifNotExists = stmt.isIfNotExists();
+        boolean setAsDefault = stmt.setAsDefault();
         StorageVault vault;
         switch (type) {
             case HDFS:
-                vault = new HdfsStorageVault(name, ifNotExists);
+                vault = new HdfsStorageVault(name, ifNotExists, setAsDefault);
```

Comment:
log if something here, it seems nothing has been logged?

Reply:
The actual rpc procedure of StorageVaultMgr would print log for message